.nvmrc

```diff
@@ -0,0 +1 @@
+v14.17.6
```

Comment:
I would recommend ensuring that all files have a final new line character.  Some parsers will ignore the last line in a file if it doesn't have a new line character on the end.  The Unix standard for a text file is that it ends in a new line character.